Dockerfile

```diff
@@ -28,7 +28,7 @@ USER default
 RUN yarn && yarn cache clean --force
 
 # Copy all necessary files
-COPY tsconfig.json .eslintignore .eslintrc.json .prettierrc.json .env /app/
+COPY index.html vite.config.ts tsconfig.json .eslintignore .eslintrc.json .prettierrc.json .env* /app/
```

Comment:
Had some trouble running with Docker, config not being able to find module `vite-plugin-eslint` 🤔

Reply:
For some reason Docker seams to get node_modules from the cache at least in a case that package is moved from devDependencies to dependencies. To be honest, I don't believe that the issue is related to this PR.

I hade to run following commands to get the missing `eslint-plugin-vitest-globals` module, which was missing in my case, to Docker image node_modules:
```
docker system prune
docker compose build
docker compose up
```

Reply:
I think the issue might be related to Docker compose volumes, but this needs little bit more investigation

Reply:
I think the problem is old node_modules which are coming from the volume of the container. To fix this you can run `docker compose down` to remove old container and network or `docker compose down -v` if you also want to remove anonymous volumes.
https://docs.docker.com/engine/reference/commandline/compose_down/

After that `docker compose up` should work normally

Reply:
Must have been the volumes, now it worked 👍